hertzbeat-collector/hertzbeat-collector-basic/src/main/java/org/apache/hertzbeat/collector/collect/database/JdbcCommonCollect.java

```diff
@@ -292,6 +292,11 @@ private String constructDatabaseUrl(JdbcProtocol jdbcProtocol) {
         if (Objects.nonNull(jdbcProtocol.getUrl())
                 && !Objects.equals("", jdbcProtocol.getUrl())
                 && jdbcProtocol.getUrl().startsWith("jdbc")) {
+            String url = jdbcProtocol.getUrl().toLowerCase(); // convert the URL to lowercase for case-insensitive checking
+            // check whether the parameter is valid
+            if (url.contains("create trigger") || url.contains("create alias") || url.contains("runscript from")) {
```

Comment:
👍 hi, these keyword `allowLoadLocalInfile` `allowLoadLocalInfileInPath` `useLocalInfile` also will cause

Reply:
> 👍 hi, these keyword `allowLoadLocalInfile` `allowLoadLocalInfileInPath` `useLocalInfile` also will cause

ok, thank you for your advice. I'll go and add it.